_layouts/day-archive.html

```diff
@@ -2,18 +2,35 @@
 layout: default
 ---
 <ol class="breadcrumb">
-	<li class="breadcrumb-item"><a href="/"><span class="typcn typcn-news"></span> News</a></li>
-	<li class="breadcrumb-item"><a href="/{{ page.date | date: "%Y" }}">{{ page.date | date: "%Y" }}</a></li>
-	<li class="breadcrumb-item"><a href="/{{ page.date | date: "%Y/%m" }}">{{ page.date | date: "%B" }}</a></li>
-	<li class="breadcrumb-item active text-truncate">{{ page.date | date: "%d" }}</li>
+	<li class="breadcrumb-item">
+		<a href="{{ site.baseurl }}/">
+			{% if site.breadcrumb_icon %}
+			<svg class="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
+				<path d="{{ site.breadcrumb_icon }}" />
+			</svg>
+			{% endif %}
+			{{ site.title_short }}
+		</a>
+	</li>
+	<li class="breadcrumb-item">
+		<a href="{{ site.baseurl }}/{{ page.date | date: '%Y' }}">
+			{{ page.date | date: '%Y' }}
+		</a>
+	</li>
+	<li class="breadcrumb-item">
+		<a href="{{ site.baseurl }}/{{ page.date | date: '%Y/%m' }}">
+			{% assign m = page.date | date: '%m' | minus: 1 %}{{ site.messages.months[m] }}
```

Comment:
You can just use `%B` instead of `%m`, that displays full month names already

Reply:
I want it to be translateable. You can add month name translations in  _config.yml